vmm/src/config.rs

```diff
@@ -276,6 +281,8 @@ impl MemoryConfig {
                 mergeable_str = &param[10..];
             } else if param.starts_with("hotplug_size=") {
                 hotplug_str = &param[13..]
+            } else if param.starts_with("virtiomem_size=") {
```

Comment:
Drop this and add `hotplug_method=[virtio-mem|acpi`]

Reply:
Make sure to update the help.

Reply:
Fixed.